packages/newspringchurchapp/src/prayer/PrayerCard/PrayerCard.tests.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Providers from 'newspringchurchapp/src/Providers';
+
+import PrayerCard from './PrayerCard';
+
+const cardData = {
+  imageSource: {
+    uri: 'https://fillmurray.com/400/600',
+  },
+  name: 'Bill',
+  prayer:
```

Comment:
I'm guessing the snapshot for this text is technically broken (even though it won't show it) because `prayer` should be `text`.
```suggestion
  text:
```